README.md

```diff
@@ -11,6 +11,71 @@
 **Interested in helping out?** Take a look at the [contributing
 doc](CONTRIBUTING.md) to find out how.
 
+## Build
+
+The operator is based on the [Operator
+SDK](https://github.com/operator-framework/operator-sdk). In order to build the
+operator, you first need to install the SDK. [Instructions are
+here.](https://github.com/operator-framework/operator-sdk#quick-start)
+
+Once the SDK is installed, Anthill can be built via:
+
+```bash
+$ dep ensure
+
+$ operator-sdk build docker.io/gluster/anthill
+INFO[0001] Building Docker image docker.io/gluster/anthill
+Sending build context to Docker daemon  114.7MB
+Step 1/4 : FROM alpine:3.8
+ ---> 196d12cf6ab1
+Step 2/4 : RUN apk upgrade --update --no-cache
+ ---> Using cache
+ ---> 6a5bb76fe272
+Step 3/4 : USER nobody
+ ---> Using cache
+ ---> faf8acce50e4
+Step 4/4 : ADD build/_output/bin/anthill /usr/local/bin/anthill
+ ---> Using cache
+ ---> b404638145da
+Successfully built b404638145da
+Successfully tagged gluster/anthill:latest
+INFO[0002] Operator build complete.
+```
+
 ## Installation
 
-Once we have something to install, it'll be easy... we promise. :smile:
+Install the CRDs into the cluster:
+
+```bash
+$ kubectl apply -f deploy/crds/operator_v1alpha1_glustercluster_crd.yaml
+customresourcedefinition.apiextensions.k8s.io "glusterclusters.operator.gluster.org" created
+```
+
+Install the service account, role, and rolebinding:
+
+```bash
+$ kubectl apply -f deploy/service_account.yaml
+serviceaccount "anthill" created
+
+$ kubectl apply -f deploy/role.yaml
+role.rbac.authorization.k8s.io "anthill" created
+
+$ kubectl apply -f deploy/role_binding.yaml
+rolebinding.rbac.authorization.k8s.io "anthill" created
```

Comment:
The roles can be combined in a single Yaml file.

Reply:
done